### ISSUE TITLE
Standards for no-author and undated material

### ISSUE THREAD

**Issue by jedbrown** [openjournals/joss#607]
It's not uncommon to see material (often other software packages) that surely should be cited and perhaps even has a citation suggestion on their websites, but where there is no date and/or no named authors.

Some list the author as "Foo Team", which results in the citation "(Team, n.d.)" and "Team, F. n.d." in the references.  This can be improved via extra braces in paper.bib:

    author = {{Foo Team}},

Some journals have citation standards that require the author field be non-empty.  I see some just add the current year on otherwise-undated web-page references. (This seems reasonable for continuously-updated software documentation.)

I believe there is some variability in how JOSS editors handle these cases and think it would be useful to specify JOSS preferences, perhaps next to the [example paper](https://joss.readthedocs.io/en/latest/submitting.html?highlight=bib#example-paper-and-bibliography).

**Comment by arfon** [comment]
Thanks for flagging this @jedbrown!

So just to be clear, are you saying if the author field has extra braces (e.g. `author = {{Foo Team}}`) then the `n.d.` doesn't show up?

**Comment by arfon** [comment]
Also, / cc https://github.com/openjournals/joss/pull/554 as this feels related.

**Comment by jedbrown** [comment]
No, but the extra braces enable `(Foo Team, n.d.)`. We have to add a date to get rid of the `n.d.`. I don't know under what circumstances `n.d.` should be allowed, versus replaced with a concrete data. I think some projects don't mention the date because they don't want to maintain it each year and/or would like citations to not fragment across versions.

**Comment by danielskatz** [comment]
I think the current example paper handles this, so I'm going to close this issue
